test/hotspot/jtreg/compiler/c2/irTests/AddINodeIdealizationTests.java

```diff
@@ -43,7 +43,9 @@ public static void main(String[] args) {
                  "test8", "test9", "test10",
                  "test11", "test12", "test13",
                  "test14", "test15", "test16",
-                 "test17", "test18", "test19"})
+                 "test17", "test18", "test19",
+                 "testXPlus_PosConMinusY_", "testXPlus_NegConMinusY_",
```

Comment:
Please make the name uniform and put them in correct order, thanks.

Reply:
Renamed, does it look good to you now? Thanks.